guides/PSCi.md

```diff
@@ -85,10 +90,10 @@ The PureScript compiler suite (i.e. the executable `purs`), unlike most compiler
 
 `purescript-psci-support` defines the `Eval` type class for this purpose. Instances of `Eval` are provided for `Show`able types, and for `Eff`, so that we can evaluate actions in the REPL. Library implementors might like to provide `Eval` instances for their own `Eff`-like types.
 
-## PSCi Without Pulp
+## PSCi Without Spago
 
 PSCi can be run directly, by specifying a list of PureScript source files as globs:
 
-    psci 'src/**/*.purs' 'bower_components/purescript-*/src/**/*.purs'
+    psci 'src/**/*.purs' 'output/**/*.purs'
```

Comment:
The command now is `purs repl`. I think we should just omit the secondary lib glob since `output` isn't generally a lib glob; it's what the compiler uses for it's build products and won't work with `purs repl`.

Reply:
Omitting the second one is still kind of problematic because it’s inevitably going to lead to a slew of module not found errors in almost any case it’s used (because the dependencies won’t be included). I’m not really sure what to do here; the way spago works means that there isn’t a single glob you can use here, you have to use `spago sources`, and I’m not sure it counts as “using the repl on its own” if you are still using Spago to assemble the command line arguments.

Reply:
I didn't notice the `psci` command being there, apologies. I attempted to make a suggested change in 807183b to address these concerns plus updated more references to the old `psci` command to either use `spago repl` or `purs repl` which I missed before. Thanks.